apps/docs/src/layouts/MainLayout.astro

```diff
@@ -64,13 +64,13 @@ const githubEditUrl = `${CONFIG.GITHUB_EDIT_URL}/${currentFile}`;
           <slot />
         </PageContent>
       </div>
-      <aside
+      <!-- <aside
         id="grid-right"
         class="w-full h-full grid-sidebar hidden h-screen sticky top-0 p-0 xl:col-start-3 xl:flex"
         title="Table of Contents"
       >
         <RightSidebar headings={headings} githubEditUrl={githubEditUrl} />
-      </aside>
+      </aside> -->
```

Comment:
✂️

Reply:
No left code behind (our code is versioned, we don't need to commenting out the code).

Reply:
Done